content/ai/concepts/generative-ai/generative-ai.md

```diff
@@ -0,0 +1,126 @@
+
+**GenAI** is the connection of **LLMs** with technologies that in addition to text results can generate digital content such as images, video, music, or code.
+
+There is exploration of going beyond digital content and use this technology for things as varied as the discovery of new molecules and 3D designs.
+
+Things are changing quickly and the impact is only just starting to being predicted and felt. Most people are expecting a step change in certain ways how we live and work.
+
+### Foundation models
+
+In the past, AI applications were very task specific. The recent technology shift has been to use large **foundation models** to drive a number of different tasks. Large amounts of unstructured and unsupervised training is used to create the **foundation model**. **LLMs** are a type of **foundation model**.
+
+The models have a generative ability and were the start of **Generative AI**. **ChatGPT** demonstrated the ability to generate and completed natural language prompts.
+
+Using the concept of **foundation models** we can leverage the vast amount of pre-trained data with a small amount of additional **tuning** and **prompting**. **Tuning** adds labels into the data. **Prompting** bridges the gap between training and our intention. While training the **LLM** is expensive, the usage referred to as **inference** is more cost effective.
+
+### The question of intelligence?
+
+Artificial Intelligence **AI** is the matching or exceeding the intelligence of a human being. Things that we often associate with intelligence such as doing math, playing chess, or remembering vast amounts of data already can be done more quickly better than a person.
+
+With **LLMs** and a technology called **diffusion** AI systems can now match or exceed other skills we often associate with intelligence such as:
+
+-Understanding and creating new text content.
+-Writing never before created stories and poems.
+-Analyzing content for tone and emotions.
+-Creating unique images and video.
+
+### Is it art?
+
+In 2022 Jason Allen’s AI generated work, Théâtre D’opéra Spatial, won first place in the digital category at the Colorado State Fair. The artist did indicate that he used the **GenAI** platform **Midjourney**. It was later discovered that the judges weren't familiar with the tool at the time of judging.
+
+The question does **GenAI** create art is now being asked. Parallels are being drawn. When photography was introduced, only painting had ever been considered art. Photography wasn't considered art and the same discussion took place. Now, few people would argue that photography can be art.
+
+Another argument is that since there is no advanced skill required from the operator. Some consider are the application of skills developed over many years. However, modern art has been about concept and technique - not necessarily skill.
```

Comment:
```suggestion
Another argument is that since there is no advanced skill required from the operator, it is not art. Some consider art as the application of skills developed over many years. However, some modern art has been about concept and technique - not necessarily skill.
```